templates/fargate.cfn.yml

```diff
@@ -177,6 +177,27 @@ Parameters:
       - prod
     ConstraintDescription: Specify either dev or prod
 
```

Comment:
This should be an optional alarm. Take a look at the DB alarms for the conditions.

Reply:
Updated in template.